go/pkg/kv/store.go

```diff
@@ -0,0 +1,24 @@
+package kv
+
+import (
+	"context"
+	"time"
+)
+
+// Store defines the interface for a key-value store with TTL support
+type Store interface {
+	Get(ctx context.Context, key string) ([]byte, bool, error)
+	Set(ctx context.Context, key string, workspaceID string, value []byte, ttl *time.Duration) error
+	Delete(ctx context.Context, key string) error
+	ListByWorkspace(ctx context.Context, workspaceID string, cursor int64, limit int) ([]KvEntry, error)
+}
```

Comment:
cc @chronark do we even need this anymore?

Reply:
no, we can delete the entire kv package

Reply:
@coderabbitai please create an issue